src/index.html

```diff
@@ -1,18 +1,31 @@
 <!doctype html>
-<html lang="en">
+<html lang="ru">
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Calendar</title>
+    <title>Календарь</title>
     <link
       rel="stylesheet"
       href="styles/index.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="calendar calendar--start-day-sun calendar--month-length-31"
```

Comment:
The class `calendar--start-day-sun` and `calendar--month-length-31` should not be hardcoded. These values should be set dynamically based on the current date or user input. This way, your calendar will be more flexible and reusable.

Reply:
This is specifically for passing tests.